tasks/install-matlab/v1/task.json

```diff
@@ -27,7 +26,7 @@
             "type": "string",
             "label": "Products",
             "required": false,
-            "defaultValue": "MATLAB Parallel_Computing_Toolbox",
+            "defaultValue": "MATLAB",
             "helpMarkDown": "Space-separated list of products to install. If a product name contains white-space characters, replace them with underscores. For the full list of available products and their names, see https://www.mathworks.com/products.html."
```

Comment:
This could be the same as what we have in GitHub Actions or CircleCI.

Reply:
Updated to match the products help text from GitHub Actions `action.yml`